### PR TITLE
Stubs for oauthlib

### PR DESCRIPTION
**Pull request by ismeta**
As title. This was generated with stubgen, with a touch of manual intervention to clean up and have it pass tests.

Working through CI builds now...

### CODE REVIEW

**Comment by exemplary-citizen** [comment]
@JukkaL

File: stubs/oauthlib/oauthlib/oauth1/__init__.pyi [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    WebApplicationServer as WebApplicationServer,
+)
+from .rfc5849.errors import (
+    InsecureTransportError as InsecureTransportError,
+    InvalidClientError as InvalidClientError,
+    InvalidRequestError as InvalidRequestError,
+    InvalidSignatureMethodError as InvalidSignatureMethodError,
+    OAuth1Error as OAuth1Error,
+)
+from .rfc5849.request_validator import RequestValidator as RequestValidator
```
Comment (ismeta) [inline stubs/oauthlib/oauthlib/oauth1/__init__.pyi:31]: Should __init__ unused imports be deleted as well? Wasn't sure if they're somehow special

File: stubs/oauthlib/oauthlib/oauth2/rfc6749/tokens.pyi [hunk 2]
```diff
@@ -0,0 +1,59 @@
+from typing import Any
+
+class OAuth2Token(dict[Any, Any]):
```
Comment (ismeta) [inline stubs/oauthlib/oauthlib/oauth2/rfc6749/tokens.pyi:3]: mypy is ok with `dict` typing here, but pyright wants type arguments. mypy won't accept `dict[..., ...]`. I noticed the maintainer's doc mentions favouring `dict` ('use builtin generics'). What's the best thing to do here - `Dict`?

File: stubs/oauthlib/oauthlib/oauth1/__init__.pyi (hunk 1 above)
Comment (srittau) [reply stubs/oauthlib/oauthlib/oauth1/__init__.pyi:31]: In `__init__.py` they usually make sense. If there is an `__all__` field, this is a good indication of what should be exported.

Comment (ismeta) [reply stubs/oauthlib/oauthlib/oauth1/__init__.pyi:31]: There isn't an `__all__` field, unfortunately. Should I leave this as-is?

Comment (srittau) [reply stubs/oauthlib/oauthlib/oauth1/__init__.pyi:31]: Sounds good.

File: stubs/oauthlib/oauthlib/oauth2/rfc6749/tokens.pyi (hunk 2 above)
Comment (srittau) [reply stubs/oauthlib/oauthlib/oauth2/rfc6749/tokens.pyi:3]: If using `dict[Any, Any]` doesn't work, use `Dict[Any, Any]`. (There are some fixes for built-in generics in mypy HEAD that haven't been released yet.)

Comment (ismeta) [reply stubs/oauthlib/oauthlib/oauth2/rfc6749/tokens.pyi:3]: Alright, thanks!

**Review by JukkaL — APPROVED** [review]
Thanks! Looks good.

**Pull request merged**